concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/10 Focus Cells Programatically.md

```diff
@@ -0,0 +1,7 @@
+You can use the following methods to focus a specific cell:
+
+- [focus](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#focuselement)     
+Allows you to set focus on a cell. You can use the [getCellElement](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#getCellElementrowIndex_dataField) method to get elements of required cells.
+
+- [editCell](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#editCellrowIndex_dataField)     
+Switches a cell to the editing state and moves the focus to this cell.
```

Comment:
The word "focus" requires no article. You can say it's abstract, uncountable. Therefore, just "moves focus":

```suggestion
Switches a cell to the editing state and moves focus to this cell.
```

P.S. I'm unsure about the action order here. I would have expected that the target cell accepts focus first and only then switches to the editing state. However, this fact can be completely unimportant in the current context.

Reply:
It doesn't matter much here since it happens almost simultaneously. Once you call this method, the cell is both in editing state and focused.

Reply:
OK. I see. In this case, I believe we should stick to the last version and be done with it.